pkg/discovery/dtofactory/container_dto_builder.go

```diff
@@ -220,15 +220,27 @@ func (builder *containerDTOBuilder) getContainerProperties(pod *api.Pod, index i
 	properties = append(properties, podProperties...)
 
 	ns := stitching.DefaultPropertyNamespace
-	attr := string(stitching.UUID)
+	idattr := string(stitching.CNTID)
+	fullidattr := string(stitching.CNTFULLID)
 	value := getContainerStitchingProperty(pod, index)
+	// Short containerID
+	if len(value) > 11 {
```

Comment:
Nit: I'd suggest to define a constant for this "11" number so we know what it means and the following number "12" can just be a "+1".

Reply:
I created a constant, thansk